vms/shift/views.py

```diff
@@ -759,54 +609,37 @@ def view_volunteer_shifts(request, volunteer_id):
         return HttpResponse(status=403)
 
 
-@login_required
-def volunteer_search(request):
-    user = request.user
-    admin = None
-
-    try:
-        admin = user.administrator
-    except ObjectDoesNotExist:
-        pass
-
-    # check that an admin is logged in
-    if not admin:
-        return render(request, 'vms/no_admin_rights.html')
-    else:
-        if request.method == 'POST':
-            form = SearchVolunteerForm(request.POST)
-            if form.is_valid():
-
-                first_name = form.cleaned_data['first_name']
-                last_name = form.cleaned_data['last_name']
-                city = form.cleaned_data['city']
-                state = form.cleaned_data['state']
-                country = form.cleaned_data['country']
-                organization = form.cleaned_data['organization']
-
-                volunteer_list = search_volunteers(
-                        first_name,
-                        last_name,
-                        city,
-                        state,
-                        country,
-                        organization
-                        )
-                return render(
-                    request,
-                    'shift/volunteer_search.html',
-                    {'form': form, 'has_searched': True, 'volunteer_list': volunteer_list}
-                    )
-        else:
-            form = SearchVolunteerForm()
-            volunteer_list = get_all_volunteers()

```

Comment:
@tapasweni-pathak That is already handled by FormView. It shows "Your search return no results!" when the form entered is invalid.